src/lib/button/_button-base.scss

```diff
@@ -62,14 +62,30 @@ $mat-mini-fab-padding: 8px !default;
   }
 }
 
-// Applies styles to buttons with backgrounds: raised, fab, and mini-fab
-%mat-raised-button {
+// Applies styles to buttons with stroke styling
+%mat-stroked-button {
   @extend %mat-button-base;
 
-  @include mat-overridable-elevation(2);
+  @include mat-overridable-elevation(0);
+
+  outline: 1px solid currentColor;
```

Comment:
Can this be done with `border` instead of `outline`? Generally outline should be avoided since things its used by the browser for focus treatment, devtools for element selection, etc.

Reply:
Done, it was initially done this way to prevent the border from changing the offset height/width of the button.